packages/components/menu/src/menu.scss

```diff
@@ -1,77 +1,126 @@
 :host {
-  --sl-menu-padding: 4px;
-  --_background: var(--sl-color-popover-background);
-  --_border: var(--_border-width) solid var(--_border-color);
-  --_border-color: var(--sl-color-popover-border);
-  --_border-width: var(--sl-menu-border-width, 1px);
-  --_border-radius: var(--sl-border-radius-popover-default);
-  --_box-shadow: var(--sl-elevation-shadow-md);
-  --_color: var(--sl-color-popover-foreground);
-  --_divider-margin-block: 4px;
-  --_padding-block: var(--sl-menu-padding);
-  --_padding-inline: var(--sl-menu-padding);
-
-  background: var(--_background);
-  border: var(--_border);
-  border-radius: var(--_border-radius);
-  box-shadow: var(--_box-shadow);
+  background: var(--sl-color-elevation-surface-raised);
+  border: var(--sl-color-elevation-border-raised) solid var(--sl-size-border-width-default);
+  border-radius: var(--sl-size-border-radius-default);
+  box-shadow: var(--sl-elevation-shadow-md);
   box-sizing: border-box;
-  color: var(--_color);
+  color: var(--sl-color-text-default);
   display: none;
   flex-direction: column;
+  gap: var(--sl-space-new-xs);
   margin: 0;
   max-inline-size: var(--sl-menu-max-inline-size, auto);
-  min-inline-size: var(--sl-menu-min-inline-size, fit-content);
+  min-inline-size: var(--sl-menu-min-inline-size, min-content);
   opacity: 0;
   overflow: visible auto;
-  padding-block: var(--_padding-block);
+  padding: var(--sl-space-new-md);
   padding-inline: 0;
```

Comment:
We have a `padding` the line above, so we don't need this padding-inline? Or maybe above it should be `padding-block`?